app/config.py

```
@@ -61,6 +61,11 @@ def DATABASE_URL(self) -> str:
 	NATS_URL: str = 'nats://nats:4222'
 	NATS_TOKEN: str | None = None
 
+	# Authentication
+	AUTH_SECRET_KEY: str = 'your_secret_key'
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to: "AUTH_SECRET_KEY"

(S105)